### PR TITLE
[5.2] Add multiple attachments (with zip compression option)

### PR DESCRIPTION
**Pull request by let-aurn**

### CODE REVIEW

File: src/Illuminate/Mail/Message.php [hunk 1]
```diff
@@ -181,6 +181,41 @@ public function attach($file, array $options = [])
     }
 
     /**
+     * Attach multiple files to the message (with Zip compression option).
+     *
+     * @param  array  $files
+     * @param  array  $options
+     * @return $this
+     */
+    public function attach_files($files, array $options = [], $should_compress = false, $archive_name = 'archive')
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/Message.php]: Please use camelCase

File: src/Illuminate/Mail/Message.php [hunk 2]
```diff
@@ -181,6 +181,41 @@ public function attach($file, array $options = [])
...
+    {
+        $compress_success = false;
+
+        // Trying to compress
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/Message.php]: no need for this comment - it's already obvious what the code does

File: src/Illuminate/Mail/Message.php [hunk 3]
```diff
@@ -181,6 +181,41 @@ public function attach($file, array $options = [])
     }
 
     /**
+     * Attach multiple files to the message (with Zip compression option).
+     *
+     * @param  array  $files
+     * @param  array  $options
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/Message.php]: phpdoc doesn't match the method dfn

**Comment by taylorotwell** [comment]
No plans to add this. Create your zip file first and then attach it using the normal methods available already.

**Pull request closed without merging**